.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve Qlib
+labels: bug
+
+---
+
+## 🐛 Bug Description
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
+
+Steps to reproduce the behavior:
+
+1.
+1.
+1.
+
+
+## Expected Behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Screenshot
+
+<!-- A screenshot of the error message or anything shouldn't appear-->
+
+## Environment
+
+ - Qlib version:
+ - Python version:
+ - OS (`Windows`, `Linux`, `MacOS`):
+ - Commit number:
```

Comment:
It would be better if we could provide script to collect these info. User could get these info with a single command.
`curl -sL https://xxxx/collect_info.py | python`
Example of this script
```python
import qlib
print(qlib.VERSION)
.......
```

Reply:
This will help both users and us to collect information

Reply:
Yeah, this might be a good solution to get qlib version. Maybe pengrong could help to upload such a file to the server.